app/src/main/java/app/nexd/android/ui/helper/requestOverview/HelperOverviewViewModel.kt

```diff
@@ -37,24 +33,32 @@ class HelperOverviewViewModel(application: Application) : AndroidViewModel(appli
     }
 
     fun getOtherOpenRequests(): LiveData<List<HelpRequest>> {
-        val observable = reload.flatMap {
-            // get all requests created by other people
-            api.helpRequestsControllerGetAll(
-                null,
-                true,
-                null,
-                true,
-                listOf(
-                    PENDING.value
-                )
-            )
-                .map { list -> list.filter { it.helpListId == null } } // TODO shouldn't be needed
-        }
+        val observable =
+            api.userControllerFindMe()
+                .flatMap { currentUser ->
+                    zipCode = currentUser.zipCode
+                    reloadNearby.flatMap { // get all requests created by other people
+                        api.helpRequestsControllerGetAll(
+                            "me",
```

Comment:
we should extract the "me" string into a constants class and reuse it from there